app/utils/config.example.js

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I don't think you meant to commit this file.

Reply:
Curses; that is the second time that has gotten pushed up. WIll remove.